app/src/main/java/com/postpc/tenq/services/SoundAwarenessService.java

```diff
@@ -0,0 +1,36 @@
+package com.postpc.tenq.services;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+import com.google.gson.Gson;
+
+public class SoundAwarenessService {
+
+    private static final String RECORD_SERVICE_PREFS = "record_service_prefs";
+    private static final String RECORDER_KEY = "recorder";
+    private final Context applicationContext;
+    private final Gson gson;
+    private final RecorderService recorderService;
+
+    public SoundAwarenessService(Context applicationContext, RecorderService recorderService) {
+        this.applicationContext = applicationContext;
+        this.gson = new Gson();
+        this.recorderService = recorderService;
+        this.recorderService.setUserSetRecorderOn(getRecorderStateFromSharedPreferences());
```

Comment:
`setUserSetRecorderOn` confusing name, what does this do?

Reply:
I have two variables - one for the user's choice and one if the recorder is on / off since if there is no Bluetooth device connected then the recorder is off although the user chose to use it